Source/ScrollScrubber.swift

```diff
@@ -28,100 +28,109 @@ func > <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
     }
 }
 
+enum ScrollType: Int {
+    case page
+    // `chapter` is only for the collection view if vertical with horizontal content is used
+    case chapter
+}
+
 enum ScrollDirection: Int {
     case none
     case right
     case left
     case up
     case down
-    
+
     init() {
         self = .none
     }
 }
 
 class ScrollScrubber: NSObject, UIScrollViewDelegate {
-    weak var delegate: FolioReaderCenter!
-    var showSpeed = 0.6
-    var hideSpeed = 0.6
-    var hideDelay = 1.0
-    
-    var visible = false
-    var usingSlider = false
-    var slider: UISlider!
-    var hideTimer: Timer!
-    var scrollStart: CGFloat!
-    var scrollDelta: CGFloat!
-    var scrollDeltaTimer: Timer!
-
-	var frame: CGRect! {
-		didSet {
-			self.slider.frame = frame
-		}
-	}
-
-    init(frame:CGRect) {
+    weak var delegate                   : FolioReaderCenter!
```

Comment:
As explained in the other thread, this was done beforehand not by us. 
Removing all `Forced Unwrapping` is not really in the scope of that pull request.